.github/workflows/ci.yml

```diff
@@ -137,6 +137,32 @@ jobs:
           python -c "from pyvista.plotting import system_supports_plotting; print('System support plotting ' + str(system_supports_plotting()))"
 
 
+  check-vulnerabilities-dev:
+    name: "Check library vulnerabilities (development mode)"
+    if: github.ref != 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check library vulnerabilities with development mode"
+        uses: ansys/actions/check-vulnerabilities@v8
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
+          python-package-name: ${{ env.PACKAGE_NAME }}
+          dev-mode: true
+
+
+  check-vulnerabilities-main:
+    name: "Check library vulnerabilities (default mode - only on main)"
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check library vulnerabilities with default mode"
+        uses: ansys/actions/check-vulnerabilities@v8
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
+          python-package-name: ${{ env.PACKAGE_NAME }}
+

```

Comment:
Maybe?

```suggestion
  check-vulnerabilities-dev:
    name: "Check library vulnerabilities"
    runs-on: ubuntu-latest
    steps:
      - name: "Check library vulnerabilities with development mode"
        uses: ansys/actions/check-vulnerabilities@v8
        with:
          python-version: ${{ env.MAIN_PYTHON_VERSION }}
          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
          python-package-name: ${{ env.PACKAGE_NAME }}
          dev-mode: github.ref != 'refs/heads/main'
```

Reply:
Interesting, I can try it. However, I am not sure having an operation in the ``dev-mode`` arg works

Reply:
If it doesn't, we could add another step and create a ``is_dev_mode`` output similar to what we have here:

https://github.com/ansys/pymapdl/blob/3cb1b93fd645401b940a881d37440458f42b2926/.github/workflows/ci.yml#L171-L172

Reply:
Best way to know is trying.. 😄 I'm not sure whether the action will recognize that boolean (I hope so!)